0chromium-imx8-aarch64/build-chromium.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+# Build Chromium
+
+set -ex
+
+export CXX=g++
+export PATH=/depot_tools:/root/ninja:$PATH
+
+clone https://chromium.googlesource.com/chromium/tools/depot_tools.git\
+cd depot_tools
+git checkout 8dd74d4f85a739ff883831960c665b0a3a24ec36
+
+mkdir /chromium
+cd /chromium
+wget -q https://commondatastorage.googleapis.com/chromium-browser-official/chromium-96.0.4664.110.tar.xz
+tar xf chromium-96.0.4664.110.tar.xz
+
+cd /root/
+cd git clone https://github.com/ninja-build/ninja.git -b v1.8.2
+cd /root/ninja
+git checkout 41156cd9c97a4e49473c473b7952f8cb7c43c56c
```

Comment:
please provide a comment why are you checking out this commit .

Reply:
Hi Caio, this is one of the best practices doing git inside the container, actually the only reason this is there is because if someone updates the v1.8.2 branch it doesn't break the build. That being said, the Dockerfile maintainer needs to come back to the Dockerfile update to the latest hash and make sure everything is working again.